<ui version="4.0" >
 <class>qikeaTopology</class>
 <widget class="QDialog" name="qikeaTopology" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tolology Entry</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/png/site_mod.png</normaloff>:/png/site_mod.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabelType_2" >
       <property name="text" >
        <string>Type</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxType" >
       <item>
        <property name="text" >
         <string>Include</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Exclude</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabelAddress" >
       <property name="text" >
        <string>Address</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAddress" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMask" >
        <string>009 . 009 . 009 . 009; </string>
       </property>
       <property name="text" >
        <string>0 . 0 . 0 . 0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabelNetmask" >
       <property name="text" >
        <string>Netmask</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNetmask" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMask" >
        <string>009 . 009 . 009 . 009; </string>
       </property>
       <property name="text" >
        <string>0 . 0 . 0 . 0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>41</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ikea.qrc" />
 </resources>
 <connections/>
</ui>
